### ISSUE TITLE
How to close a popup programatically?

### ISSUE THREAD

**Issue by cyrilf** [indoorequal/vue-maplibre-gl#72]
Let's say I have a popup inside a marker and I want to set `closeButton` to `false` (in order to create my own closing action).
What is the way to close this popup?

```
<mgl-marker :coordinates="coordinates">
  <mgl-popup ref="popup">
    <div>
      <div class="text-2xl">Title</div>
      <p>HTML content</p>
      <button @click="closePopup">Close</button>
    </div>
  </mgl-popup>
</mgl-marker>
```

```
<script lang="ts" setup>

const popup = useTemplateRef('popup');

// But here the `remove` function is `undefined` as it's not exposed.
const closePopup = () => popup.value?.remove();
</script>
```

I can see that the [default close button in the base library](https://github.com/maplibre/maplibre-gl-js/blob/87486a5ef2085e600e8fa4e31252629dd8488dcd/src/ui/popup.ts#L554C18-L554C30) is using a private method `_onClose` which then calls the `remove` method.

Do you have any idea how to programmatically close the popup?

**Comment by francois2metz** [comment]
Hi,

Thanks for reporting this issue. 
We could expose the remove function to outside to fix that.

**Comment by francois2metz** [comment]
Fixed in the v7.6.0 version.

**Comment by cyrilf** [comment]
Hey, thanks a lot for your reactivity! :tada:
